sensor/scripts/run_ci_tests.sh

```diff
@@ -0,0 +1,2 @@
+pytest -m "ci" --cov=src --cov=cli tests
```

Comment:
You could combine all those test scripts into one and use a command line option to choose what to run, personal opinion though 😄